### ISSUE TITLE
Figure/image paths with spaces inconsistently handled with typst

### ISSUE THREAD

**Issue by njericha** [quarto-dev/quarto-cli#10542]
### Bug description

Adding attributes to inserted images that have spaces in their path cause `%20` to be inserted when using the Typst format. This does not happen when there are no attributes, nor when using the PDF format. This causes Typst to report a `file not found` error when compiling. This also does not look like a pandoc issue since I cannot reproduce the bug [here](https://pandoc.org/try/?params=%7B%22text%22%3A%22---%5Cnformat%3A%5Cn++typst%3A%5Cn++++keep-typ%3A+true%5Cn---%5Cn%5Cn%21%5B%5D%28test.png%29%5Cn%5Cn%21%5B%5D%28test+test.png%29%5Cn%5Cn%21%5B%5D%28test.png%29%7Bwidth%3D80%25%7D%5Cn%5Cn%21%5B%5D%28test+test.png%29%7Bwidth%3D80%25%7D%22%2C%22to%22%3A%22typst%22%2C%22from%22%3A%22markdown%22%2C%22standalone%22%3Afalse%2C%22embed-resources%22%3Afalse%2C%22table-of-contents%22%3Afalse%2C%22number-sections%22%3Afalse%2C%22citeproc%22%3Afalse%2C%22html-math-method%22%3A%22plain%22%2C%22wrap%22%3A%22auto%22%2C%22highlight-style%22%3Anull%2C%22files%22%3A%7B%7D%2C%22template%22%3Anull%7D). RE: https://github.com/quarto-dev/quarto-cli/issues/3225#issuecomment-2294317097

### Steps to reproduce

````qmd
---
format:
  typst:
    keep-typ: true
---

![](test.png)

![](test test.png)

![](test.png){width=80%}

![](test test.png){width=80%}
````

### Expected behavior

The Typst file that is produced (with the first 273 lines of preamble omitted) should be

```typ
#box(image("test.png"))

#box(image("test test.png"))

#box(width: 80%,image("test.png"))

#box(width: 80%,image("test test.png"))
```

### Actual behavior

```typ
#box(image("test.png"))

#box(image("test test.png"))

#box(width: 80%,image("test.png"))

#box(width: 80%,image("test%20test.png"))
```

Swapping the format to
````qmd
---
format:
  pdf:
    keep-tex: true
---
````

produces the tex file (with the first 155 lines of preamble omitted)

```tex
\begin{document}


\includegraphics{test.png}

\includegraphics{test test.png}

\includegraphics[width=0.8\textwidth,height=\textheight]{test.png}

\includegraphics[width=0.8\textwidth,height=\textheight]{test test.png}




\end{document}
```

### Your environment

- VSCode 1.92.1
- Windows 11

### Quarto check output

```bash
Quarto 1.5.56
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.5.56
      Path: C:\Program Files\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Tex:  (not detected)

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.11.9
      Path: C:/Users/Nicholas/AppData/Local/Programs/Python/Python311/python.exe
      Jupyter: 5.7.2
      Kernels: julia-1.10, python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

**Comment by mcanouil** [comment]
Thanks for the report.

I can reproduce on 1.5.56 but not on the current development version or latest pre-release (1.6.5).

I'm going ahead and close this as completed.
Feel free to comment if it's not resolved when using 1.6.5.

**Comment by cderv** [comment]
For reference, duplicate of 
- https://github.com/quarto-dev/quarto-cli/issues/10181

which was indeed solved for 1.6
